src/c_api/c_api_executor.cc

```diff
@@ -560,6 +560,7 @@ int MXExecutorReshape(int partial_shaping,
                       ExecutorHandle *out) {
   MXAPIThreadLocalEntry *ret = MXAPIThreadLocalStore::Get();
   API_BEGIN();
+  *out = nullptr; // ensure we can know whether to free executor on early abort
```

Comment:
Small linting issue:
src/c_api/c_api_executor.cc:563:  At least two spaces is best between code and comments  [whitespace/comments] [2]
Category 'whitespace' errors found: 1

Reply:
I've added the space, and am also creating a temporary pointer for the new executor as I see is done in other functions like MXExecutorGetOptimizedSymbol.